File: solutions/kotlin/pangram/1/src/main/kotlin/Pangram.kt
object Pangram {

    fun isPangram(input: String): Boolean {
        val normalized = input.lowercase().replace(Regex("[^a-z]"), "").toSortedSet()
        return normalized == "abcdefghijklmnopqrstuvwxyz".toSortedSet()
    }
}

File: solutions/go/pangram/1/pangram.go
package pangram

import "unicode"

func IsPangram(input string) bool {
	counter := map[rune]int{}
	for _, letter := range input {
		if !unicode.IsLetter(letter) {
			continue
		}

		if _, ok := counter[unicode.ToLower(letter)]; ok {
			counter[unicode.ToLower(letter)] += 1
		} else {
			counter[unicode.ToLower(letter)] = 1
		}
	}
	return len(counter) == 26
}
